requirements.txt

```diff
@@ -1,9 +1,6 @@
 jax[cpu]
-ipython>=8.22.0
-matplotlib>=3.7.0
-numpy>=1.26.0
+jaxlib==0.4.22
```

Comment:
This seems overly strict. Should we rather use `jaxlib>=0.4.22` or even `jaxlib>=0.4`?

Reply:
That's fair, I was trying to have this work with jax-metal but in practice that isn't possible at this point.